docs/kqueen.rst

```diff
@@ -52,7 +52,9 @@ Serializers
 Configuration
 --------------
 
-Example configuration files are located in `config/` directory. Default configuration file is `config/dev.py` and this can be configured by `KQUEEN_CONFIG_FILE` environment variable. Any environment variable matching name `KQUEEN_*` will be loaded and merged with configuration file.
+Example configuration files are located in the `config/` directory. Default configuration file is `config/dev.py`.
+To set up a different configuration file set `KQUEEN_CONFIG_FILE` environment variable. Also any environment variable
+matching name `KQUEEN_<config_parameter>` will be loaded and replaced form value in the configuration file.
```

Comment:
This sentence doesn't feel right. Maybe: "Set environment variable matching pattern `KQUEEN_<config_parameter>` to override values defined in config file"

Reply:
Not sure I understand the second sentence. Might be one of the following:
* To define a different configuration file, set the `KQUEEN_CONFIG_FILE` environment variable. Any environment variable matching `KQUEEN_<config_parameter>` will be overridden by the value in the configuration file.
* To define a different configuration file, set the `KQUEEN_CONFIG_FILE` environment variable. To override the values defined in the configuration file, set the environment variable matching the KQUEEN_<config_parameter> pattern.